### ISSUE TITLE
Consider replacing the StickyToolbarView with the BalloonPanelView

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5-ui#202]
In https://github.com/ckeditor/ckeditor5-editor-inline/issues/10 I realized, that since we have `BPV#pin()` + complex positioning function, the `StickyToolbarView` probably becomes obsolete and, like the `InlineEditorUIView's` toolbar, it can be easily handled by the `BalloonPanelView`.

**Comment by Reinmar** [comment]
Like a bottom of the stack which is never removed?

Will that cause the toolbar to disappear if something else opens? Is it a good idea?

**Comment by oleq** [comment]
> Will that cause the toolbar to disappear if something else opens? Is it a good idea?

No, no. I'm talking about a separate BPV instance, which has nothing to do with ContextualBallon, which handles Link, Image, etc. Two things next to each other.

**Comment by Reinmar** [comment]
So, what would this change improve?

**Comment by oleq** [comment]
1. This code https://github.com/ckeditor/ckeditor5-ui/blob/master/src/toolbar/sticky/stickytoolbarview.js#L111-L130 and this https://github.com/ckeditor/ckeditor5-ui/blob/master/src/toolbar/sticky/stickytoolbarview.js#L189-L214 will be reduced to just a couple of lines.
2. Same positioning engine means less code branching and easier maintenance.
2. Same positioning engine means that each integration benefits from bug fixes done to the engine.
